selfdrive/locationd/test/test_locationd.py

```diff
@@ -0,0 +1,165 @@
+#!/usr/bin/env python3
+import os
+import json
+import random
+import unittest
+import time
+from cffi import FFI
+
+from cereal import log
+import cereal.messaging as messaging
+from common.params import Params
+
+from selfdrive.manager.process_config import managed_processes
+
+SENSOR_DECIMATION = 10
+VISION_DECIMATION = 2
+
+LIBLOCATIOND_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__), '../liblocationd.so'))
+
+random.seed(123489234)
```

Comment:
Put this in the `setUp()`? Now it runs at import time, so other things can use the RNG inbetween.